.github/ISSUE_TEMPLATE/documentation_update.yml

```diff
@@ -1,13 +1,9 @@
 name: Documentation Update
-description: Describe this issue found in the documentation
-title: "[Doc]: "
-labels: ["📖 documentation", "🔍 triage"]
-assignees: ["florentmaitre", "paulinea"]
+description: Please fill out the following details for your documentation update request.
```

Comment:
This description appears in that dialog:

<img width="961" alt="Capture d’écran 2025-04-29 à 10 04 25" src="https://github.com/user-attachments/assets/712a5300-dc9e-4162-a0fa-d6363b27fe07" />

Thus I don't think the sentence should start with "Please fill out the following details". Maybe something such as "Propose  a documentation change" or similar.

Same comment for "demo_app_update.yml" and "component_update.yml".